api/db-cdr/generate-cdr/cb-criteria-refactor/make-cb-criteria-01-proc-cpt4.sh

```diff
@@ -0,0 +1,221 @@
+#!/bin/bash
```

Comment:
These numbers are related to the ID range of the rows generated by the file. @arychj suggested I make the ID range a parameter to avoid collisions (right now they are hardcoded). So I will remove the number from the filenames and add a parameter for the id range.

Reply:
I have removed the numbers from filenames and implemented a parameter for the ID range.